src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -81,14 +109,23 @@ export const setSelectedSpecies = createStandardAction(
   'species_selector/species_selected'
 )<SearchMatch | PopularSpecies>();
 
+export const clearSearchResults = createAction(
+  'species_selector/clear_search_results',
+  (resolve) => {
+    return () =>
+      resolve(
+        undefined,
+        buildAnalyticsObject({
+          category: categories.POPULAR_SPECIES,
```

Comment:
Erm. Why is it using the PopularSpecies category?

Reply:
Also, it seems we don't need to track this action at all.

Reply:
Yes, it shouldn't be. Not sure why I left it there. Removed it now.